stric/README.md

```diff
@@ -0,0 +1,7 @@
+This is the project bootstrapped using `@stricjs/kit`.
```

Comment:
If possible, please use the following template to match the convention, I think you can add some additional detail after the template, can you confirm? @Kapsonfire-DE 

> # Stricjs with Bun runtime
> 
> ## Getting Started
> To get started with this template, simply paste this command into your terminal:
> ```bash
> bun create strictjs./strictjs-example
> ```
>
> ## Development
> To start the development server run:
> ```bash
> bun run dev
>```
> 
> Open http://localhost:3000/ with your browser to see the result.

Reply:
that would be fine 👍

Reply:
Added :)